src/App.tsx

```diff
@@ -1,24 +1,69 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React,
+{
+  useContext, useEffect, useMemo,
+} from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import * as todoService from './api/todos';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorType } from './types/ErrorType';
+import { ErrorInfo } from './components/ErrorInfo/ErrorInfo';
+import { filterTodos } from './helpers/filterTodos';
+import { USER_ID } from './utils/userId';
+import { AppContext } from './contexts/appContext';
 
 export const App: React.FC = () => {
+  const {
+    todosFromServer,
+    filterBy,
+    setTodosFromServer,
+    setErrorMessage,
+  } = useContext(AppContext);
+
+  useEffect(() => {
+    todoService.getTodos(USER_ID)
+      .then(setTodosFromServer)
+      .catch(() => setErrorMessage(ErrorType.UnableToLoad));
+  }, [setTodosFromServer, setErrorMessage]);
+
+  const todosToView = useMemo(
+    () => filterTodos(todosFromServer, filterBy),
+    [todosFromServer, filterBy],
+  );
+
+  const isEveryTodosCompleted = todosToView.every(
+    todo => todo.completed,
+  );
+
+  const uncompletedTodosCount = todosFromServer
+    .filter((todo) => !todo.completed).length;
+
+  const isSomeTodosCompleted = todosFromServer.some((todo) => todo.completed);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-add-and-delete#react-todo-app-add-and-delete">React Todo App - Add and Delete</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+
+        <Header isEveryTodosCompleted={isEveryTodosCompleted} />
```

Comment:
```suggestion
      <div className="todoapp__content">
        <Header isEveryTodosCompleted={isEveryTodosCompleted} />
```
Don't leave empty lines between parent and child elements